### DISCUSSION TITLE
Suggestion for species ID

### DISCUSSION THREAD

**Discussion by ChrisMoExp** [Ildaron/Laser_control#2] [Ideas]
Can I suggest using a directional microphone system to capture the sound of the insect target and determine from the frequency composition which species the insect is.

this would enable target preferencing to avoid bees and other non evil insects.

**Comment by Ildaron** [comment]
> Can I suggest using a directional microphone system to capture the sound of the insect target and determine from the frequency composition which species the insect is.
> 
> this would enable target preferencing to avoid bees and other non evil insects.

thank you, sounds great

**Reply by Oferedavid131** [reply]
Nice

**Comment by Oferedavid131** [comment]
🥰

**Comment by Ildaron** [comment]
> Can I suggest using a directional microphone system to capture the sound of the insect target and determine from the frequency composition which species the insect is.
> 
> this would enable target preferencing to avoid bees and other non evil insects.

yes, I did it 
https://www.researchgate.net/publication/349226713_Machine_vision_for_low-cost_remote_control_of_mosquitoes_by_power_laser

**Comment by roerichard888-debug** [comment]
A directional microphone system could be an interesting approach for identifying insect targets based on their sound frequency. This could improve target selection and help avoid beneficial insects like bees. For professional pest control in Portsmouth, technologies like this could potentially support more precise and environmentally responsible treatments.
Website:https://newhampshirepestpros.us/locations/pest-control-in-portsmouth
